scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -780,10 +829,38 @@ class ContainerChecker( StoppableThread ):
         Monitors containers to check when they start and stop running.
     """
 
-    def __init__( self, config, global_config, logger, socket_file, docker_api_version, host_hostname, data_path, log_path,
+    def __init__( self, config, global_config, logger, socket_file, docker_api_version, agent_pod, host_hostname, log_path,
                   include_all, include_controller_info, namespaces_to_ignore,
                   ignore_pod_sandboxes ):
+        """
 
+        :param config: The configuration for the monitor which includes options for the container checker
```

Comment:
Nitpick should this be `@param` instead of `:param`.  Same for all following params.